release/RELEASE.md

```diff
@@ -182,145 +210,157 @@ This testing procedure starts once the release is listed on the [releases page].
    > The ad-hoc machines take a bit of time to be available after being reported as
    > created, so be patient for a bit if your first connection attempt(s) fail.
    >
-   > NOTE 1: **Use Firefox for testing.** Windows machines come with both Internet Explorer and Firefox installed. Do not make the mistake of trying to use Internet Explorer.
+   > At this point, use Firefox to download and install `daml-sdk-$VERSION-windows.exe` from https://github.com/digital-asset/daml/releases
+   > (please ensure `$VERSION` is expanded correctly!.
+   > 
+   > NOTE 1: **Use Firefox for testing.** Windows machines come with both Internet Explorer and Firefox installed. Do 
+   > not make the mistake of trying to use Internet Explorer.
    >
    > Ad-hoc machines also come with Node, VSCode and OpenJDK preinstalled, so
    > you don't need to worry about those.
    >
-   > NOTE 2: After logging in, **it takes some time for the machine to be configured.** The script that installs Firefox, Node, VSCode and OpenJDK runs once the
-   > machine is available for login. The software you need should appear within about 10 minutes. (An easy way to check is to try to open `D:\` , as it is created after all the software is installed.)
+   > NOTE 2: After logging in, **it takes some time for the machine to be configured.** The script that installs Firefox, 
+   > Node, VSCode and OpenJDK runs once the machine is available for login. The software you need should appear within 
+   > about 10 minutes (an easy way to check is to try to open `D:\` , as this volume is created after all the software 
+   > is installed).
    >
-   > All of the commands mentioned in this document can be run from a simple
+   > All of the commands mentioned in this testing section can be run from a simple
    > DOS prompt (start menu -> type "cmd" -> click "Command prompt").
+   > 
+   > At the end of your Windows testing session, please be sure to terminate the GCP instance by running 
+   > `./ad-hoc.sh destroy $ID`. Here `$ID` is the identity for your GCP instance - this is printed when you create your 
+   > Windows instance.
```

Comment:
Thanks for that :-)

Reply:
Perhaps we should recommend `temp` instead of `create`?

Reply:
Good point! I keep forgetting that exists.